spec/jsonapi/authorization_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+RSpec.describe JSONAPI::Authorization::DefaultPunditAuthorizer do
```

Comment:
I suppose these specs actually don't test `DefaultPunditAuthorizer` so this describe argument is misleading, don't you think? And if you decide to go the way of moving `user_context` method to `Configuration` class as commented in https://github.com/venuu/jsonapi-authorization/pull/32/files#r76201264, these tests could also be moved to `spec/jsonapi/authorization/configuration_spec.rb` file.